server/live_query/redis_live_query_test.go

```diff
@@ -0,0 +1,129 @@
+package live_query
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/kolide/fleet/server/kolide"
+	"github.com/kolide/fleet/server/pubsub"
+	"github.com/kolide/fleet/server/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRedisLiveQuery(t *testing.T) {
+	if _, ok := os.LookupEnv("REDIS_TEST"); !ok {
+		t.SkipNow()
+	}
+
+	for _, f := range testFunctions {
```

Comment:
Ah, good catch. The test functions are actually made to be agnostic to the live query store type (using the kolide.LiveQueryStore interface) -- similar to the datastore tests. I rearranged the files and naming so that this makes more sense.